data/json/items/tool_armor.json

```diff
@@ -392,6 +392,285 @@
       }
     ]
   },
+  {
+    "id": "ar_glasses_t0",
+    "type": "TOOL_ARMOR",
+    "category": "clothing",
+    "name": { "str": "cheap pair of AR glasses", "str_pl": "cheap pairs of AR glasses" },
+    "description": "A pair of cheap augmented reality glasses.  There is a camera at the rim, and it also has an alarm feature.",
+    "weight": "100 g",
+    "volume": "200 ml",
+    "price": 1000,
+    "price_postapoc": 250,
+    "to_hit": -2,
+    "material": [ "glass", "plastic" ],
+    "symbol": "[",
+    "looks_like": "glasses_safety",
+    "color": "light_gray",
+    "covers": [ "eyes" ],
+    "coverage": 40,
```

Comment:
Are they really so slim so they cover only 40% of eyes?

Reply:
At a moment the cheap one were envisionned as covering only one eye. I will up the coverage to 95.